### PR TITLE
feat(RAN-25): Сделать рекурсивный обход awesome-list'ов

### PR DESCRIPTION
**Pull request by DeadlySquad13**
Closes issue [RAN-25]

# Summary
Сделал обход с помощью фетча репозиториев из гитхаба по топику `awesome-list`.

[RAN-25]: https://repo-analyzer.atlassian.net/browse/RAN-25

### CODE REVIEW

**Review by RDcool — COMMENTED** [review]
1. Отсутствуют комментарии в tests/awesome_list_miner/test_md_parser.py ?
2.  Зачем нужен пустой import в src/awesome_list_miner/main.py ?

File: src/awesome_list_miner/__main__.py [hunk 1]
```diff
@@ -1,4 +1,4 @@
-from projen_template.example import hello
+from awesome_list_miner.example import hello
```
Comment (Nbslab) [inline src/awesome_list_miner/__main__.py]: Как в MR по RAN-26, не понимаю цель использования функции hello() и этого скрипта в с ним в совокупности

**Comment by DeadlySquad13** [comment]
> 1. Отсутствуют комментарии в tests/awesome_list_miner/test_md_parser.py ?

Если что-то конкретное в этом файле непонятно, давай разберем это отдельно.

Не оставил комментарии, так как это тесты. По TDD это все равно, что документацию комментировать:
https://www.guru99.com/test-driven-development.html
Они лучше любого описания код комментируют, ведь ты сразу видишь, как использовать код и corner cases, когда написанный код ломается / не должен ломаться.

Я в целом не ставил целью ко всем файлам комментарии оставлять, так как опыт показывает, что комментарии устаревают быстрее кода. Лучше любых слов это видео пояснит:
https://www.youtube.com/watch?v=Bf7vDBBOBUA

> 2. Зачем нужен пустой import в src/awesome_list_miner/main.py ?

Спасибо, я вообще файл уберу.

File: src/awesome_list_miner/main.py [hunk 2]
```diff
@@ -0,0 +1 @@
+import
```
Comment (Nbslab) [inline src/awesome_list_miner/main.py]: Не понял, зачем используется пустой скрипт с импортом

**Review by Nbslab — COMMENTED** [review]
Оставил 2 комментария по игре(цели использования двух скриптов, она собственно и не ясна)

**Review by Nbslab — APPROVED** [review]
Молодец, жду в овервотче

**Review by RDcool — APPROVED** [review]
Стас, красава
